### ISSUE TITLE
RetentionPeriod: 999y is considered less than 1 day

### ISSUE THREAD

**Issue by thomas725** [VictoriaMetrics/VictoriaMetrics#6330]
### Describe the bug

Today I've failed at upgrading my single-server (raspberry pi4 debian) VM v1.90 to v1.101, because the new version failed to start with the same cli parameters I've used with 1.90.

### To Reproduce

Download, extract and start "victoria-metrics-linux-arm64-v1.101.0.tar.gz" with the specified parameter and observe the fatal error that prevents startup.

### Version

victoria-metrics-20240425-145433-tags-v1.101.0-0-g5334f0c2c

### Logs

```
fatal        VictoriaMetrics/app/vmstorage/main.go:104        -retentionPeriod cannot be smaller than a day; got 999y
```

### Used command-line flags

`-retentionPeriod=999y`

**Comment by thomas725** [comment]
okey, I found a workaround, reducing the retention period from 999y to 99y makes the newer version accept the cli parameter. Probably the system won't live that long anyway ;)

**Comment by hagen1778** [comment]
@thomas725 https://github.com/VictoriaMetrics/VictoriaMetrics/pull/6486 has been merged and will be included into the next release. Thanks for reporting the issue!

**Comment by valyala** [comment]
FYI, VictoriaMetrics should check for the maximum value passed to `-retentionPeriod` command-line flag starting from [v1.102.0-rc2](https://github.com/VictoriaMetrics/VictoriaMetrics/releases/tag/v1.102.0-rc2).

**Comment by valyala** [comment]
FYI, the bugfix for this issue has been included in [v1.97.6 LTS release](https://github.com/VictoriaMetrics/VictoriaMetrics/releases/tag/v1.97.6).
